actions/commands.py

```diff
@@ -148,33 +150,129 @@ def check_status(request):
-#     except Exception as e:
-#         message = e.__str__()
+    # make a list if statuses and check if new status is valid
+    for status in status_results:
+        title = status.get("title")
+        statuses_list.append(title)
 
+    if new_status not in statuses_list:
+        client.chat_postEphemeral(
+            channel=channel_id,
+            text=f"Invalid status, choose from: {statuses_list}",
+            user=data.get("user_id")
+        )
+        return HttpResponse(status=404)
+
+    # if status valid get the id 
+    for status in status_results:
+        if status.get("title") == new_status:
+            new_status_id = status.get("id")
+            print(new_status_id)
+
+    # update the status id
+    try:
+        response = auth_req.patch(
+            url=f"http://127.0.0.1:8000/api/teams/{team_id}/tickets/{ticket_id}/", 
+            data={"status": int(new_status_id)}
+            )
+        message = json.dumps(response.json(), indent=4)
+    except Exception as e:
+        message = e.__str__()
+
+    if response.status_code != 200:
+        client.chat_postEphemeral(
+            channel=channel_id,
+            text="Error, try again",
+            user=data.get("user_id")
+        )
+        return HttpResponse(status=404)
 
-#     response = requests.post(
-#         url="http://127.0.0.1:8000/ticket/{}/".format(args.get("id")),
-#         data={"status_id": args.get("status")})
+    print(response.status_code)
+    response = auth_req.get(url=f"http://127.0.0.1:8000/api/teams/{team_id}/tickets/{ticket_id}/").json().get("status")
+    print(response)
     
-#     client.chat_postMessage(
-#         channel = channel_id
-#         text = "Ticket status updated!\n"
-#     )
-#     return HttpResponse(status=200)
+    client.chat_postMessage(
+        channel = channel_id,
+        text = "Ticket status updated!"
+    )
+    return HttpResponse(status=200)
+
+@csrf_exempt
+def print_statuses(request):
+    data = request.POST
+    channel_id = data.get("channel_id")
+    text = data.get("text")
+    args = ArgumentParser.parse_args(text)
+    team_id = args.get("team_id")
+
+    user_id = data.get("user_id")
+    auth_req = AuthorizedRequest(user_id=user_id)
+
+    try:
+        response = auth_req.get(url=f"http://127.0.0.1:8000/api/teams/{1}/statuses/")
+        message = json.dumps(response.json(), indent=4)
+    except Exception as e:
+        message = e.__str__()
+
+    if response.status_code != 200:
+        client.chat_postEphemeral(
+            channel=channel_id,
+            text="Error, try again",
+            user=data.get("user_id")
+        )
+        return HttpResponse(status=404)
+
+    print(message)
+
+    status_results = response.json().get("results")
+    status_message = f"Team {1} statuses: "
+
+    for status in status_results:
+        title = status.get("title")
+        status_message += f"{title}, "
+
+    status_message = status_message[:-2]
```

Comment:
hardcoding indexes is not the best since status_results could be empty. prefer to use join() function.